### PR TITLE
Fix cli unpack

### PR DESCRIPTION
**Pull request by Shukla-Gaurav**

### CODE REVIEW

**Review by PhaneeshB — APPROVED** [review]

File: apps/language_models/scripts/vicuna.py [hunk 1]
```diff
@@ -408,7 +408,7 @@ def generate_new_token(self, params, sharded=True, cli=True):
             _past_key_values = output["past_key_values"]
             _token = int(torch.argmax(_logits[:, -1, :], dim=1)[0])
         else:
-            print(len(output))
+            #  print(len(output))
```
Comment (dan-garvey) [inline apps/language_models/scripts/vicuna.py:411]: can be deleted

**Review by dan-garvey — DISMISSED** [review]

**Review by PhaneeshB — APPROVED** [review]
LGTM

File: requirements.txt [hunk 2]
```diff
@@ -19,7 +19,7 @@ parameterized
 
 # Add transformers, diffusers and scipy since it most commonly used
 transformers
-diffusers
+diffusers==0.19.3
```
[the remark this answers is not in the document]
Comment (PhaneeshB) [reply requirements.txt:22]: ohh, looks like 0.20.0 fails for train lora SD

[the remark this answers is not in the document]
Comment (Shukla-Gaurav) [reply requirements.txt:22]: The latest one is crashing in LoRA train module while launching the web studio. Will unpin once the issue is fixed.

**Pull request merged**